src/style.css

```diff
@@ -1 +1,36 @@
-/* styles go here */
+.label-first-form-bottom {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.second-form {
+  margin-top: 20px;
+}
+
+.label-second-form-bottom {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.third-form {
+  margin-top: 20px;
+}
+
+.label-third-form-bottom {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.select-cars {
+  overflow-y: hidden;
+}
+
+.last-form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.label-last-form-bottom {
+  display: block;
+  margin-bottom: 10px;
+}
```

Comment:
Rewrite styles and classes. For now, you have a lot of duplicated styles. Try to improve it

Reply:
Done